src/index.html

```diff
@@ -7,6 +7,8 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Any hard-coded links/scripts should omit the protocol and just use `//fonts.googleapis.com......` so it works in HTTP and HTTPS without warnings or errors.